mcstatus/protocol/connection.py

```diff
@@ -220,7 +219,7 @@ def write(self, data: Union[bytes, bytearray]) -> None:
     def __del__(self):
         try:
             self.socket.close()
-        except:
```

Comment:
#53 has the history on this. `AttributeError`

Reply:
Should these exceptions really be handled from this PR? Check thos note I've placed in the PR\s description:
> All of the bare excepts found by flake8-bugbear were only addressed by making them except Exception, which doesn't really solve the underlying issue of not ignoring other exceptions and only excepting those which we absolutely need to, however since I don't know what these statements were supposed to be excepting, **and this isn't a place to fix that**, I've merely added an inline TODO comment mentioning this for each of these excepts.

I don't mind fixing it here, but I'd think it should be done from another PR later, this PR is just about adding flake8, not about fixing what cleanup PR didn't finish and making these except statements specific. Should I really address these in here?